sympy/integrals/laplace.py

```diff
@@ -989,6 +989,88 @@ def _laplace_apply_simple_rules(f, t, s):
     return None
 
 
+def laplace_correspondence(f, fdict):
```

Comment:
Maybe the parameter can be documented https://docs.sympy.org/latest/contributing/docstring.html#parameters-section

And also it should be better to pick up more better name than `fdict`, or make the the parameter positional-only
https://peps.python.org/pep-0570/ to make it anonymous

For example, it is possible to call like `laplace_correspondence(f=..., fdict=...)`

Reply:
Oh ... I did not know that there are positional-only arguments. Very useful!